core/sail/shacl/src/test/java/org/eclipse/rdf4j/sail/shacl/TransactionSettingsTest.java

```diff
@@ -167,6 +167,32 @@ public void testDefaultOverride() {
 			assert transactionSettings.getValidationApproach() == ShaclSail.TransactionSettings.ValidationApproach.Auto;
 			assert !transactionSettings.isCacheSelectNodes();
 			assert !transactionSettings.isParallelValidation();
+			assert transactionSettings.getIsolationLevel() == IsolationLevels.SNAPSHOT_READ;
```

Comment:
Wrong kind of assert. These should be junit or assertj assertions, not java asserts.